jwerty.js

```diff
@@ -191,7 +191,8 @@
             period: 190,
             'full-stop': 190,
             // Slash, or /, or forward-slash
-            '/': 191,
+            // '/': 191,
```

Comment:
`191` is the `/` character on the main keyboard. By removing it, we lose the ability to catch the main keyboard's `/`.

Perhaps we could name the numpad one `'num/'`, `'num-/'`or `'num-slash'`